JDBC/IssueBookPOJO.java

```diff
@@ -0,0 +1,11 @@
+package com.metacube.jdbc.example;
+
+public class IssueBooks {
```

Comment:
ACTION: class name should not be pluralize,Please  Refer http://www.iwombat.com/standards/JavaStyleGuide.html#Class%20and%20Interface%20Names for class and other java naming conventions.